apps/vx-scan/backend/test/helpers/app_helpers.ts

```diff
@@ -37,45 +34,11 @@ import {
 } from '../../src/interpret';
 import { createWorkspace, Workspace } from '../../src/util/workspace';
 
-export function get(app: Application, path: string): request.Test {
-  return request(app).get(path).accept('application/json').expect(200);
-}
-
-export function patch(
-  app: Application,
-  path: string,
-  body?: object | string
-): request.Test {
-  return request(app)
-    .patch(path)
-    .accept('application/json')
-    .set(
-      'Content-Type',
-      typeof body === 'string' ? 'application/octet-stream' : 'application/json'
-    )
-    .send(body)
-    .expect((res) => {
-      // eslint-disable-next-line no-console
-      if (res.status !== 200) console.error(res.body);
-    })
-    .expect(200, { status: 'ok' });
-}
-
-export function post(
-  app: Application,
-  path: string,
-  body?: object
-): request.Test {
-  return request(app)
-    .post(path)
-    .accept('application/json')
-    .send(body)
-    .expect((res) => {
-      // eslint-disable-next-line no-console
-      if (res.status !== 200) console.error(res.body);
-    })
-    .expect(200, { status: 'ok' });
-}
+// TODO(jonah) - Is there a way to ensure Grout always has access to node-fetch
+// in a node environment?
```

Comment:
If we're talking about tests you could add it to the Jest test setup, i.e. a [`setupFiles` entry](https://jestjs.io/docs/configuration#setupfiles-array).

Reply:
Will address in a follow up: https://github.com/votingworks/vxsuite/issues/2858